docs/getting-started/provision-infrastructure.md

```diff
@@ -1,258 +1,333 @@
 ---
 title: Provision Infrastructure
 toc: true
-weight: 3
+weight: 2
```

Comment:
It looks like "Install & Configure" is also at `weight: 2`. Should it be at `weight: 1` to ensure it appears before this?